src/scss/pages/_privacyPolicy.scss

```diff
@@ -0,0 +1,71 @@
+// Privacy Policy Styles
+
+
+// .privacy-policy {
```

Comment:
Think we can get rid of this commented out section and a lot of the empty lines in this file